dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/ds3treetable/Ds3PutJob.java

```diff
@@ -160,10 +158,14 @@ public void executeJob() throws Exception {
                     totalSent.addAndGet(l);
                 });
                 job.attachObjectCompletedListener(obj -> {
+                    final Calendar currentTime = Calendar.getInstance();
```

Comment:
The new java 8 time libraries should be used rather than the old pre java 8 time libraries.